components/src/core/UserMenu/UserMenu.tsx

```diff
@@ -186,22 +186,22 @@ const UserMenuRender: React.ForwardRefRenderFunction<unknown, UserMenuProps> = (
                 classes: {
                     ...aProps.classes,
                     root: cx(
-                        defaultClasses.avatarRoot,
-                        preserveOnClick ? '' : defaultClasses.noCursor,
+                        generatedClasses.avatarRoot,
+                        preserveOnClick ? '' : generatedClasses.noCursor,
                         aProps?.classes?.root
                     ),
                 },
             });
         },
-        [avatar, onOpen, defaultClasses, classes]
+        [avatar, onOpen, generatedClasses, classes]
```

Comment:
I suspect this isn't being used
```suggestion
        [avatar, onOpen, generatedClasses]
```